tendermint-rs/src/public_keys.rs

```diff
@@ -112,70 +107,11 @@ impl Deref for TendermintKey {
     }
 }
 
-/// Secret Connection signing keys
-// TODO(tarcieri): unify with `TendermintKey`?
-#[derive(Copy, Clone, Debug, Hash, Eq, PartialEq)]
-pub enum SecretConnectionKey {
```

Comment:
Moved and renamed this to `tendermint::secret_connection::PublicKey`. This seems better with `secret_connection` as (presently) an optional cargo feature, however I just noticed we don't test `tendermint-rs` with `--no-default-features`